### PR TITLE
Feature/add reference input

### PR DESCRIPTION
**Pull request by k-y0u**
Add new type of input : Reference !

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -47,5 +47,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Update dependencies
 - Add Save button in ModelizerNavigationBar
 - Add onUpdateFile function in ModelizerTextView and create FileTabHeader component
+- Add RefrenceInput
```
Comment (Zorin95670) [inline changelog.md:50]: cool be better and without type :/

File: tests/unit/components/inputs/ReferenceInput.spec.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            }],
+          },
+        },
+      },
+    });
+  });
+
+  describe('Test variables initialization', () => {
+    describe('Test prop: attribute', () => {
+      it('Should be an object containing name & value', () => {
```
Comment (justinehell) [inline tests/unit/components/inputs/ReferenceInput.spec.js:36]: why not testing `attribute.definition` ? (same for plugin object)

**Review by justinehell — CHANGES_REQUESTED** [review]

File: tests/unit/components/inputs/ReferenceInput.spec.js (hunk 2 above)
Comment (k-y0u) [reply tests/unit/components/inputs/ReferenceInput.spec.js:36]: if I didn't make mistakes, I'm only testing the direct usages.
`attribute.value` become a `ref`, `attribute.name` is a `label` in the template and `plugin.name` is used in the `q-icon :name`.
The other used parts of attribute and plugin are check with the refs `options` and `iconName`.
So no need to duplicate. Please, notify me if I'm missing something.

File: changelog.md (hunk 1 above)
Comment (k-y0u) [reply changelog.md]: done

**Review by justinehell — APPROVED** [review]

**Review by justinehell — DISMISSED** [review]

**Review by Zorin95670 — APPROVED** [review]

**Review by justinehell — APPROVED** [review]

**Pull request merged**